tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
```

Comment:
Yes, I tried it at the beginning when I started the migration to grunt and I couldn't get it to work. But now when everything is building and test are passing it was much easier :)